docs/examples/word_embedding/word_embedding_training.md

```diff
@@ -109,30 +113,16 @@ fastText embedding model can compute meaningful embedding vectors for tokens
 that were not seen during training.
 
 For this notebook, we have prepared a helper function `transform_data_fasttext`
-which builds a series of transformations of the `text8` `Dataset` created above,
-applying "tricks" mentioned before. It returns a `DataStream` over batches as
-well as a batchify_fn function that applied to a batch looks up and includes the
-fastText subwords associated with the center words and finally the subword
-function that can be used to obtain the subwords of a given string
+which builds a series of transformations of the `text8 Dataset` created above,
+applying the techniques we mention briefly above. It returns a `DataStream` over batches as
+well as a `batchify_fn` function that applied to a batch looks up and includes the
+fastText subwords associated with the center words. Additionally, it returns the subword
+function which can be used to obtain the subwords of a given string
 representation of a token. We will take a closer look at the subword function
-shortly.
+farther on.
 
-Note that the number of subwords is potentially
-different for every word. Therefore the batchify_fn represents a word with its
-subwords as a row in a compressed sparse row (CSR) matrix. Take a look at
-https://mxnet.incubator.apache.org/tutorials/sparse/csr.html if you are not
-familia with CSR.  Separating the batchify_fn from the previous word-pair
-sampling is useful, as it allows to parallelize the CSR matrix construction over
-multiple CPU cores for separate batches.
-
-You can find it in `data.py` in the
-archive that can be downloaded via the Download button at the top of this page.
-- [1] Mikolov, Tomas, et al. “Distributed representations of words and phrases
-and their compositionally.”
-   Advances in neural information processing
-systems. 2013.
-- [2] Bojanowski et al., "Enriching Word Vectors with Subword
-Information" Transactions of the Association for Computational Linguistics 2017
+You can find the transform_data_fasttext in `data.py` in the
```

Comment:
the `transform_data_fasttext()` function

Reply:
How do I amend this? Do i have to do a new pull?

Reply:
you can push new commits to the same branch.